src/components/Input.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Input = (props) => {
+
+    return (
+            <fieldset>
+                <label htmlFor={props.id}>{props.label}</label>
+                <input
+                    id={props.id}
```

Comment:
`props.id` is being repeated three times, please assign a variable to the value or destructure `props`